runtime/parachains/src/dmp/tests.rs

```diff
@@ -205,3 +209,69 @@ fn verify_dmq_mqc_head_is_externally_accessible() {
 		);
 	});
 }
+
+#[test]
+fn verify_fee_increment_and_decrement() {
+	let a = ParaId::from(123);
+	let mut genesis = default_genesis_config();
+	genesis.configuration.config.max_downward_message_size = 16777216;
+	new_test_ext(genesis).execute_with(|| {
+		let default = InitialFactor::get();
```

Comment:
Can we rename this to `initial`? `default` is actually a contextual keyword in Rust, let's not abuse it here.